app/src/main/java/com/github/h3lp3rs/h3lp/database/repositories/Repository.kt

```diff
@@ -3,27 +3,26 @@ package com.github.h3lp3rs.h3lp.database.repositories
 import java.util.concurrent.CompletableFuture
 
 /**
- * General interface displaying the different methods required to store, retrieve and delete an object of T in the database
+ * General interface displaying the different methods required to store, retrieve and delete an
+ * object of T in the database
  */
-interface Repository<T>{
+interface Repository<T> {
```

Comment:
Same as above. Would increase coverage as well I believe. This class does not score **A** on code climate and is  kind of useless.